typespec/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "type": "module",
   "dependencies": {
-    "@typespec/compiler": "latest",
-    "@azure-tools/typespec-autorest": "latest",
-    "@azure-tools/typespec-azure-core": "latest",
-    "@azure-tools/typespec-azure-resource-manager": "latest",
-    "@azure-tools/typespec-providerhub": "latest",
-    "@typespec/http": "latest",
-    "@typespec/openapi": "latest",
-    "@typespec/rest": "latest",
-    "@typespec/versioning": "latest"
+    "@typespec/compiler": "^0.60.0",
```

Comment:
I read this as meaning we are ok with any version from `0.60.0` up to, but not including, `1.0.0`. Is that correct?

Reply:
I think you're right. These should be `~0.60.0`

Reply:
this would be more restrictive then. I thought typically minor versions are meant to be backward compatible but this approach would mean greater control in how we handle updates.

Reply:
More restrictive is good. Semver basically has no rules for `0.X.Y` versions.